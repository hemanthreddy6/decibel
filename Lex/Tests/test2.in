if a%b {
    a *= b;
} or a/c {
    a %= c;
} otherwise {
    a /= d;
}