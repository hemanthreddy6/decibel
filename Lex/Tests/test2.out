IF
IDENTIFIER
%
IDENTIFIER
{
IDENTIFIER
*=
IDENTIFIER
;
}
OR
IDENTIFIER
/
IDENTIFIER
{
IDENTIFIER
%=
IDENTIFIER
;
}
OTHERWISE
{
IDENTIFIER
/=
IDENTIFIER
;
}
